tests/e2e/local/minikube/README.md

```diff
@@ -13,14 +13,15 @@
   * [curl](https://curl.haxx.se/) - Verify `curl --help` prints the help information.
 
 You can run the following script to check/install of all pre-requisites, or use it as a reference to install them manually.
-(This requires installation of [Homebrew](https://brew.sh) on macOS or debian based Linux distributions)
+(Only Debian is supported at this point.)
```

Comment:
Did something change such that we don't support mac?

Reply:
At that point, only Debian installation script is available.